### DISCUSSION TITLE
Is there a way to export query logs?

### DISCUSSION THREAD

**Discussion by AxisRay** [TechnitiumSoftware/DnsServer#1100] [Q&A]
<img width="1167" alt="image" src="https://github.com/user-attachments/assets/0742706a-ba78-4b5a-9268-909f157f0b61">
I want to export the query logs for analysis. Are there any methods to achieve this?

**Comment by AxisRay** [comment]
Closed
I exported the necessary data from the sqlite database.

**Reply by ShreyasZare** [reply]
You can use the HTTP API too to pull the data in JSON format.
